inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -230,15 +230,49 @@ def simulate(
         resource_pool_id: typing.Optional[uuid.UUID] = None,
         run_async: bool = False,
         plasma_surface_filepath: typing.Optional[types.Path] = None,
+        num_iterations: int = None,
+        num_samples: int = None,
+        sigma_scaling_factor: float = None,
     ) -> tasks.Task:
         """Simulates the scenario.
 
+        A simulation can be performed in two ways simply by adding some more
+        arguments. 
+        
+        If `num_iterations`, `num_samples` and `sigma_scaling_factor`
+        are all not provided, the simulation merely uses the created device and 
+        performs the computation of the magnetic field produced by its coils
+        on the given plasma surface. Additionally, it produces an output of
+        the objective functions that assess the quality of the device.
+
+        Alternatively, if the arguments are provided, the simulation process
+        is different. It also uses the create device, but as a beginning step.
+        Using this initial device, the simulation adds noise to the coefficients
+        that describe the coils using a normal distribution function, generating
+        `num_samples` configurations. After this, the configuration with the
+        lowest value of the objective functions is used as an initial step for
+        the next iteration, being that this process is repeated `num_iterations`
+        times. All the generated configurations (their Fourier Series 
```

Comment:
"... used as an initial step for the next iteration. This process is repeated \`num_iterations\` times."